nuxt.config.ts

```diff
@@ -14,4 +14,8 @@ export default defineNuxtConfig({
   imports: {
     dirs: ['composables/api/**'],
   },
+  // EXPERIMENTAL: https://nitro.unjs.io/config/#routerules
+  routeRules: {
+    '/profile': { redirect: '/profile/bookings' },
+  },
```

Comment:
I'm curious - is there a non-experimental alternative for such redirects in Nuxt? 🤔

Reply:
Something like this Nuxt v2 has, but it's not working on V3... I added it, I know that it's experimental, but in the future, it will work fine